### DISCUSSION TITLE
Summer vacation 🌞

### DISCUSSION THREAD

**Discussion by Taitava** [Taitava/obsidian-shellcommands#414] [Announcements]
Hi all! 👋  Just a short heads up: Lately the development of this plugin has not been so frequent (which is fine), and I see it won't be active anytime soon. I just started my vacation and as I'm working as a programmer, I also also need to avoid hobby programming projects to be able to really rest and do things that are not part of my typical everyday life.

The _Shell commands_ plugin is a big project and still has a lot of things to improve (and also a lot of things that work well already), so I thought it's better to actively communicate about the currently passive development phase. 🙂👍 

If you do have problems, questions, feature ideas etc., please continue posting discussions about them! It might take time for me to answer, though.

**Comment by chrisgurney** [comment]
Enjoy your time off! Thank you for all the personal time you've spent on this super-useful plugin! I know I referred to your code more than once for examples, while I built my own plugin. So, thanks again for your contributions!

**Reply by Taitava** [reply]
Thank you so much for your kind words! 😍 Also nice to hear that my code has been a useful reference for developing your own plugin! 🚀

**Reply by jvsteiner** [reply]
I second this - its incredibly well written.
